ozaria/site/components/common/LayoutAspectRatioContainer.vue

```diff
@@ -25,8 +25,10 @@
     },
 
     mounted () {
-      window.addEventListener('resize', this.onResize)
-      this.onResize()
+      this.$nextTick(() => {
```

Comment:
I changed this because on initial page load the dimensions of parent component were 0 in `onResize`, and updated correctly when resized window after the initial load. Whereas calling `onResize` after nextTick, correctly gives the dimensions even on initial page loading. `nextTick` is fired after the entire view is rendered(https://vuejs.org/v2/api/#mounted), which is when I believe we want to call `onResize`.

Reply:
Ok, I think the changes requested above make sense then.  This should allow us to cover all of the cases.

Something else to think about: what you're describing seems to indicate that the parent container is waiting to receive its size from its children in some way (or the styles that expand it to the width / height of it's parent are being applied later).  Either way the top level containers here should be able to have deterministic heights / widths (100% width / 100% height of their parent in some way).  There may be an opportunity to improve the styles here.

Either way we should keep this change, but this improvement may also prevent render flickers / jumpy resizes in the future.

Reply:
Yep, definitely a room for improvement for the overall styling.
Will update this for now. Thanks!